tools/schema_loader.cc

```diff
@@ -41,6 +41,8 @@
 
 namespace {
 
+logging::logger sllog("schema_loader");
```

Comment:
Although logging is better than nothing, so I am not against this patch - I don't think "normal" users will want to use it, and will prefer regular error messages saying exactly what's wrong - not intermixed with a dozen other messages about what went right.

Reply:
A agree with you. The logging here is for tests and developers diagnosing the bug. By default, none of these logs are shown, they are all debug and trace.
The next patch is making sure we get a user-readable error in a case we got a cryptic message before.